openmetadata-ui/src/main/resources/ui/src/components/MyData/MyDataWidget/MyDataWidget.component.tsx

```diff
@@ -282,7 +384,7 @@ const MyDataWidgetInternal = ({
                 className="view-more-text text-sm font-regular m-b-lg cursor-pointer"
                 data-testid="view-more-link"
                 to={getUserPath(currentUser?.name ?? '', 'mydata')}>
-                {t('label.view-more-capital')}{' '}
+                {t('label.view-more-count', { count: data.length })}
```

Comment:
Let's use the common widget wrapper, footer, sort, etc, for MyDataWidget as well.